packages/core/src/transport/sendWithRetryStrategy.ts

```diff
@@ -133,7 +134,10 @@ function retryQueuedPayloads(
 }
 
 function shouldRetryRequest(response: HttpResponse) {
-  return response.status === 0 || response.status === 408 || response.status === 429 || response.status >= 500
+  return (
+    response?.type !== 'opaque' &&
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
    response.type !== 'opaque' &&
```